public/app/plugins/panel/table/TablePanel.tsx

```diff
@@ -26,6 +38,33 @@ export function TablePanel(props: Props) {
 
   let tableHeight = height;
 
+  const onRowHover = useCallback(
+    (idx: number, frame: DataFrame) => {
+      if (!panelContext.sync || panelContext.sync() === DashboardCursorSync.Off) {
+        return;
+      }
+
+      if (!hasTimeField(frame)) {
+        return;
+      }
+
+      const timeField: Field = frame!.fields.find((f) => f.type === FieldType.time)!;
+
+      panelContext.eventBus.publish(
```

Comment:
It feels a bit complex to have logic split between internal Table component and the outer TablePanel.

Can't it all be on handled inside the table component, any benefit to having the subscribe there and the publish here ?

Reply:
I agree, there's no point in having this here. I must've overlooked this bit of code after the table breakdown. Will move it accordingly.